internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1225,6 +1225,11 @@ func (c *ClusterManager) setKafkaPerClusterCountMetrics() error {
 		return err
 	} else {
 		for _, counter := range counters {
+			// ignore counts that do not have the cluster id set as they'll err when retrieving the cluster external id with not found
```

Comment:
either the kafkas count includes:
- rejected kafkas but not re-assigned
- or accepted kafkas but have not assigned in an OSD cluster